cfgov/unprocessed/css/organisms/carousel.less

```diff
@@ -242,18 +272,21 @@
 
   // Revert item layout.
   &_items {
-    width: auto;
     display: block;
+    width: auto;
+    height: auto;
   }
 
   &_item {
-    position: static;
     display: block;
+    width: auto;
+    height: 300px;
```

Comment:
I noticed that content can overflow the text container below 900px on `.no-js` browsers.

I'm wondering, could we set the height on `.o-carousel_item-visual` instead of `.o-carousel_item` in the `.no-js` context? The image appears to still resize in webkit browsers with JS turned off, and while it remains 300px in IE8 and IE9 even when the text grows vertically, that might be an acceptable compromise.

Reply:
Yep, that fix works, and I agree that it's an acceptable compromise. Updating!

Reply:
Updated and force-pushed with a rebase.